.storybook/preview-head.html

```diff
@@ -0,0 +1,29 @@
+<style type="text/css">
+@font-face {
+    font-family: '72';
+    src: url('./72-Regular.woff') format('woff');
```

Comment:
This is stupid and annoying, but I couldn't get the fonts from @sap-theming to load from node_modules in here 😢